bears/kotlin/KotlinLintBear.py

```diff
@@ -0,0 +1,27 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.DistributionRequirement import (
+    DistributionRequirement)
+
+
+@linter(executable='ktlint',
+        global_bear=True,
+        output_format='regex',
+        output_regex=r'(?P<filename>.*):(?P<line>\d+):'
+                     r'(?P<column>\d+): (?P<message>.+)')
+class KotlinLintBear:
+    """
+    Lints your Kotlin files.
+    Checks for coding standards or semantical problems that might lead
```

Comment:
semantical --> semantic
sorry, didn't look at it before :laughing: 
Also, you can consider removing *that might lead to problems in execution of code*.

Reply:
change or --> and